src/Microsoft.AspNetCore.Hosting/Internal/WebHost.cs

```diff
@@ -118,6 +121,26 @@ public virtual void Start()
             _hostedServiceExecutor.Start();
 
             _logger.Started();
+
+            // REVIEW: Is this the right place to log these errors?
```

Comment:
Is there a bug logged for this? I'm not a fan of REVIEW/TODO comments in checked-in code (unless they point to an open issue).